### PR TITLE
Another bug fix

### PR DESCRIPTION
**Pull request by henryjiang-sfgov**
When sms or email failed to sent, it stops the app from processing subsequent records.  This is related to the other bug:
https://github.com/SFDigitalServices/vaccination-distribution/pull/31

### CODE REVIEW

**Review by cschlessinger — APPROVED** [review]
Wait on an engineer's approval before merging

File: service/resources/notifications.py [hunk 1]
```diff
@@ -46,14 +46,19 @@ def send_notifications(self, notification_type, data, sub_type=None):
         phone_status = email_status = False
 
         if user_phone and sms_opt_in:
-            phone_status = self.send_sms(data, notification_type, sub_type)
-
+            try:
+                phone_status = self.send_sms(data, notification_type, sub_type)
+            except Exception: # pylint: disable=broad-except
+                pass
```
Comment (josh-chou) [inline service/resources/notifications.py:52]: Do we want to log the error instead of just consuming the error?  Maybe a simple print statement.

**Review by josh-chou — CHANGES_REQUESTED** [review]

File: service/resources/notifications.py (hunk 1 above)
Comment (henryjiang-sfgov) [reply service/resources/notifications.py:51]: print statement will ignore the pass statement and stops execution of the app.

Comment (henryjiang-sfgov) [reply service/resources/notifications.py:51]: If we have time, we can implement a logger.  There's not much we can do about these errors except logging them, app execution needs to continue.

Comment (josh-chou) [reply service/resources/notifications.py:52]: you can print and then pass.
printing will show up in heroku logs.

Comment (cschlessinger) [reply service/resources/notifications.py:52]: Let's make a separate PR to do that

**Pull request merged**